docs/guides/advanced/node_red.md

```diff
@@ -0,0 +1,112 @@
+# OXRS StateMonitor Node Red integration
```

Comment:
Wonder if this page should be titled `OXRS Node-RED Integrations` with sub-sections for each firmware type we provide examples for?

Same for your other Home Assistant page?